### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by sheldygg**

### CODE REVIEW

File: sqladmin/templates/sqladmin/base.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet" href="{{ url_for('admin:statics', path='css/select2.min.css') }}">
   <link rel="stylesheet" href="{{ url_for('admin:statics', path='css/flatpickr.min.css') }}">
   <link rel="stylesheet" href="{{ url_for('admin:statics', path='css/main.css') }}">
+  {% if admin.logo_url %}
+    <link rel="icon" type="image/png" href="{{ admin.logo_url}}"/>
```
Comment (aminalaee) [inline sqladmin/templates/sqladmin/base.html:13]:
```suggestion
    <link rel="icon" href="{{ admin.logo_url}}"/>
```

**Review by aminalaee — COMMENTED** [review]
Thanks for the PR, just a minor comment

**Comment by aminalaee** [comment]
I just realized we already have: https://github.com/aminalaee/sqladmin/blob/32637d15960ba22065774390acac9f6da0be86be/sqladmin/templates/sqladmin/layout.html#L9-L11

Have you faced any issues with that?

**Comment by sheldygg** [comment]
> I just realized we already have:
> 
> https://github.com/aminalaee/sqladmin/blob/32637d15960ba22065774390acac9f6da0be86be/sqladmin/templates/sqladmin/layout.html#L9-L11
> 
> Have you faced any issues with that?

No, your code add logo_url to page but no to favicon, this is problem which solve my PR

**Review by aminalaee — APPROVED** [review]
👍

**Pull request merged**